core/build.gradle.kts

```diff
@@ -47,8 +47,26 @@ android {
     buildConfigField("long", "VERSION_CODE", "${generateVersionCode()}")
   }
   buildTypes {
+    getByName("debug") {
+      dependencies {
+        implementation(Libs.objectbox_kotlin)
+        implementation(Libs.objectbox_rxjava)
+      }
+    }
     getByName("release") {
       isMinifyEnabled = false
+      apply(plugin = "io.objectbox")
+      dependencies {
+        implementation(Libs.objectbox_kotlin)
+        implementation(Libs.objectbox_rxjava)
+      }
+    }
+    create("playStore") {
+      apply(plugin = "io.objectbox")
```

Comment:
here too.

Reply:
@MohitMaliFtechiz those are for the different build types for different stores.

Reply:
@gouri-panda Okay .

* But we are adding this plugin on top of ```core/build.gradle``` (By this way it's include this plugin in every variant). https://github.com/kiwix/kiwix-android/blob/563170cd76c9b6f2e8a0562dbefd89850e551249/core/build.gradle.kts#L19
So why we are adding this again in ```release``` , ```playstore``` variant?

* We are removing the ```io.objectbox``` dependency from fdroid build, so is it necessary to add ```apply(plugin = "io.objectbox")``` in ```release``` , ```playstore``` variants?
```
"fdroidImplementation"(
    Libs.objectbox_kotlin,
  ) {
    exclude("io.objectbox")
  }
```

Reply:
@MohitMaliFtechiz Done https://github.com/kiwix/kiwix-android/pull/3143/commits/2b95ad2bf155b2a9a781115dd07a795fad904cfa. 

>We are removing the io.objectbox dependency from fdroid build, so is it necessary to add apply(plugin = "io.objectbox") in release , playstore variants?

Yes!, in order to use the Objectbox dependency we should add the plugin as well.